pages/common/sdiff.md

```diff
@@ -0,0 +1,23 @@
+# sdiff
+
+> Compare the differences and/or merge between two files.
+
+- Compare 2 files:
+
+`sdiff {{path/to/first_file}} {{path/to/second_file}}`
```

Comment:
Let's change first_file to file1 and second_file to file2 everywhere to make things short and simple.

Reply:
I'm using file1 and file 2 at first and then think first_file and second_file might be better. What do you think? @sbrl

Reply:
I'll change as your suggest since I also use file1 and file2 in another PR.